config/endpoints/api_entreprise/5_inpi_rne_beneficiaires.yml

```diff
@@ -0,0 +1,42 @@
+---
+- uid: 'inpi/rne/beneficiaires_effectifs'
+  path: '/v3/inpi/rne/unites_legales/{siren}/beneficiaires_effectifs'
+  position: 300
+  perimeter:
+    entity_type_description: |+
+      Toutes les entreprises inscrit au RCS (Registre du Commerce et des Sociétés) ainsi qu'au RNM (Répertoire Nationale des Métiers)
```

Comment:
@skelz0r est-ce que l'API utilisée est celle correspondant à cette fiche : https://www.inpi.fr/beneficiaires-effectifs ?
Si oui, il semblerait qu'il s'agisse uniquement des BE déclarés au RCS, d'où tires-tu l'information que l'API concerne aussi le RNM ? (je vois effectivement qu'il y a des observations du RNM ...)

Reply:
Non je tire ça de l'API RBE.
J'ai conjecturé avec un script d'exploration que j'avais bien des BE du RNM.

Reply:
Top super